doc/api.md

```diff
@@ -567,6 +568,28 @@ which is `details.internal_name` and falls back to `title`.
 - `document_type` *(required)*
   - The `document_type` value that returned editions has.
 
+## `GET /v2/links/changes`
+
+Returns an array of changes to links, giving the information on what
+changed (source and target content ids, and the link type), when the
+change happened, what user was associated with the action, and if the
+link was created or deleted.
+
+The results will be in decending order by link change id, which
+approximates to newer changes coming before older changes. A maximum
+of 250 changes will be returned.
+
+### Query string parameters
+
+- `link_types[]` *(required)*
+  - Filter the changes by link type.
+- `source_content_ids[]`
+  - Filter the changes by source content id.
+- `target_content_ids[]`
+  - Filter the changes by source content id.
+- `users[]`
+  - Filter the changes by user UIDs.
```

Comment:
As these don't have to be arrays it might be more appropriate to specify them as non arrays and suggest they can optionally be arrays.

Though I'm not sure how this has been handled in other parts of this doc and it's probably best to just be consistent.

Reply:
Not sure, I think not using arrays here would be confusing. Does it matter that you technically can send single values?

Reply:
Nah, that's not a problem.